Docs: document that the Flickable intercepts the event (#9070)
slint-ui/slint: docs/astro/src/content/docs/reference/gestures/flickable.mdx

   a `TouchArea`, then `Flickable` will flick immediately on pointer move events when the euclidean distance
   to the coordinates of the press event exceeds 8 logical pixels.

All pointer and mouse events that occur within the bounds of a `Flickable` are intercepted by the `Flickable`

> itself and are not propagated to any elements underneath it.

Reviewer: What do you think about this clarification? I feel we tend to speak about the element hierarchy in the context of propagation, instead of the visual order.

```suggestion
itself and are not propagated to the parent element.
```
